.github/scripts/sink_cache.py

```diff
@@ -0,0 +1,235 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+# CLI to Delete Github Action Cache.
+#
+# This script remove Cache entries a specified Github repository.
+# This allow future workflow to run on clear cache data (since they'll have to recreate them)
+import argparse
+from datetime import datetime
+from functools import reduce
+import json
+import logging
+import subprocess
+from typing import Any, Dict, Optional
+from logging import basicConfig, getLogger
+
+import sys
+from os.path import dirname, abspath
+
+current_dir = dirname(abspath(__file__))
+parent_dir = dirname(dirname(current_dir))
+sys.path.insert(0, parent_dir)
```

Comment:
Adding import location this way should be avoided whenever possible (it's basically a hack that lead to unexpected import dependencies)

Here you need it to get `bench.naturalsize` that is only use for a logging.info... so here just stick to KISS:
```python
f"Will remove entry {entry.key} at {ref or '<no_ref>'} (~{entry.size_in_bytes / 2**20:.3f}Mo)"
```

Reply:
I've refactor the code to not use `naturalize`